lighthouse-core/test/results/artifacts/artifacts.json

```diff
@@ -1757,6 +1757,12 @@
       "name": "jQuery",
       "version": "2.1.1",
       "npm": "jquery"
+    },
+    {
+      "detector": "js",
```

Comment:
these artifacts come from running LH against `dbw_tester.html`, and while we don't update this file often, it is pretty brittle to have to manually add this in here.

I'm not sure what's better/worse...adding it in the generation script or [making it look like `dbw_tester.html` is served by wordpress](https://github.com/johnmichel/Library-Detector-for-Chrome/blob/68a527631db0a256932b001e17e8e173e591b55a/library/libraries.js#L1369-L1370).

Reply:
Yeah either way its not great 😦 we could make an example stack pack that only dbw_tester passes on?  or make a flag to manually run stack packs on the cli?

Reply:
not sure what others think, but I think my vote would be to make it look like `dbw_tester.html` was a wordpress page.

Seems like matching `script[src*="wp-includes"]` would be the easiest way to pass, just making sure to pick a MIME type that won't do anything (like [`'example'`](https://tools.ietf.org/html/rfc4735)). So could add e.g.

```html
<!-- Add non-functional script to appear to be a Wordpress page for Stack Packs. -->
<script type="example" src="wp-includes"></script>
```